src/pg/sql/40_observatory_utility.sql

```diff
@@ -152,6 +179,25 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+-- Function that replaces all non digits or letters with _ trims and lowercases the
+-- passed measure name
+
+CREATE OR REPLACE FUNCTION cdb_observatory._OBS_NormalizeMeasureName(measure_name text )
```

Comment:
Realizing "normalize" conflicts with the way we work with denominators... howzabout calling this `_OBS_StandardizeMeasureName()` instead?